src/events/cd-event-sessions.vue

```diff
@@ -78,22 +67,31 @@
       totalBooked() {
         return this.children.length;
       },
+      applications() {
+        return [].concat(...(this.children).map(child => child.value));
```

Comment:
map already returns an array, no need to concat. If it doesn't (because this.children is empty?) you can safely default it to `map() || []`

Reply:
Would https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/entries help then instead ?

Reply:
I don't think so from what i have tried with it